submissions/ayodeleayobami/javascript/product.js

```diff
@@ -0,0 +1,37 @@
+var products=[
+    {
+        name:"sneakers",
+        image:"images/shoe1.jpg",
+        price:"$150"
+    },
+    {
+        name:"derby",
+        image:"images/shoe2.jpg",
+        price:"$150"
+    },
+    {
+        name:"pumps",
+        image:"images/shoe3.jpg",
+        price:"$120"
+    },
+];
+function loadProducts() {
+    products.forEach(function(item) {
+          product =
```

Comment:
you need to declare the variable `product` here. e.g. `var product`. you can also declare it outside the loop. also the lines from `<div class ...` to `</div>` should be wrapped as a string before you can assign to the variable `product`.  Make sure to test properly

See